lib/maths/CDecompositionComponent.cc

```diff
@@ -245,6 +254,10 @@ bool CDecompositionComponent::CPackedSplines::acceptRestoreTraverser(
         this->interpolate(knots, values, variances, boundary);
     }
 
+    LOG_DEBUG(<< "knots = " << core::CContainerPrinter::print(knots));
+    LOG_DEBUG(<< "values = " << core::CContainerPrinter::print(values));
+    LOG_DEBUG(<< "variances = " << core::CContainerPrinter::print(variances));
```

Comment:
Do we need this? There is trace logging in `this->interpolate` above and I don't think we want this extra debug logging.